CHANGELOG.md

```diff
@@ -130,6 +130,15 @@ now alternatively be binary-encoded instead.
 
 - `PostAndParseWithRetry` error logging now includes log URI in messages.
 
+### Minimal Gossip Example
+
+All the code for this, except for the x509ext package has been moved over
```

Comment:
comma after "package"?

Reply:
Done + the same in the gossip/README.md.